src/components/EarningAreaChart/TooltipContent.tsx

```diff
@@ -0,0 +1,220 @@
+import { Trans } from '@lingui/macro'
+import { darken, rgba } from 'polished'
+import { useEffect, useMemo } from 'react'
+import { Flex, Text } from 'rebass'
+import styled from 'styled-components'
+
+import Logo, { NetworkLogo } from 'components/Logo'
+import useTheme from 'hooks/useTheme'
+import { EarningStatsTick } from 'types/myEarnings'
+import { formattedNum } from 'utils'
+
+const formatUSDValue = (v: number) => {
+  if (v === 0) {
+    return '$0'
+  }
+
+  if (v < 0.0001) {
+    return '< $0.0001'
+  }
+
+  const formatter = Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    notation: 'compact',
+    maximumSignificantDigits: 4,
+  })
+
+  return formatter.format(v)
+}
+
+const TokensWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 8px 16px;
+  color: ${({ theme }) => theme.subText};
+  font-weight: 500;
+`
+
+const formatTokenAmount = (a: number | string) => {
+  return formattedNum(String(a), false)
+}
```

Comment:
`src/utils/index.ts/formattedNum`

Reply:
this is a wrap of formattedNum to reuse second params 
I guess that's Hung's intent. and it's ok to me